test/e2e_node/init_containers_test.go

```diff
@@ -44,28 +44,50 @@ const (
 	containerTypeRegular
 )
 
-type containerTestConfig struct {
-	Name     string
-	Type     containerType
+type execCommand struct {
 	ExitCode int
 	Delay    int
 }
 
-func (c containerTestConfig) Command() []string {
+type containerTestConfig struct {
+	Name     		string
+	Type     		containerType
+	Container		execCommand
+	PostStart*	execCommand
```

Comment:
adding this is not ideal. I think I want to rewrite framework, the real Pod will be constructed in the test and then we will have a helper function that will add all the volumes needed to every container. @tzneal wdyt?

Reply:
next thing we may need to add are various restart policies, probes, etc.

Reply:
Volume mounting is removed in the other PR, but yeah as-is it's a bit limited in terms of what it can do.  I'm supportive of extending to make testing other things easier.